src/test/java/org/opensearch/security/dlic/dlsfls/DlsTest.java

```diff
@@ -251,6 +259,32 @@ public void testDls() throws Exception {
 
     }
 
+    @Test
+    public void testDlsWithTermsQuery() throws Exception {
+
+        setup();
+
+        HttpResponse res;
+
+        Assert.assertEquals(
+            HttpStatus.SC_OK,
+            (res = rh.executeGetRequest("/terms/_search?pretty", encodeBasicHeader("dept_manager", "password"))).getStatusCode()
+        );
+        Assert.assertTrue(res.getBody().contains("\"value\" : 1,\n      \"relation"));
```

Comment:
Nit; Switch to `res.getTextFromJsonBody(...)` instead of this fragile `res.getBody().contains(...)`  check(s)

Reply:
I had to modify the RestHelper used in this test to add those methods. I copied the implementation from here: https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java#L347-L349